packages/api/src/beacon/routes/beacon/pool.ts

```diff
@@ -112,6 +142,22 @@ export type Endpoints = {
    * If one or more attestations fail validation the node MUST return a 400 error with details of which attestations have failed, and why.
    */
   submitPoolAttestations: Endpoint<
+    "POST",
+    {signedAttestations: AttestationListPhase0},
+    {body: unknown},
+    EmptyResponseData,
+    EmptyMeta
+  >;
+
+  /**
+   * Submit Attestation objects to node
+   * Submits Attestation objects to the node.  Each attestation in the request body is processed individually.
+   *
+   * If an attestation is validated successfully the node MUST publish that attestation on the appropriate subnet.
+   *
+   * If one or more attestations fail validation the node MUST return a 400 error with details of which attestations have failed, and why.
+   */
+  submitPoolAttestationsV2: Endpoint<
```

Comment:
no need to copy the headers of `submitPoolAttestations`, just note "same to the above" and clarify the difference here. Same to other v2 methods.

Reply:
Slightly in favor of just copying the jsdoc unless there is a good way to reference it somehow, as a consumer won't be looking at the v1 api. The difference of the apis should be clear based on types.

Reply:
> no need to copy the headers of `submitPoolAttestations`, just note "same to the above" and clarify the difference here. Same to other v2 methods.

The endpoint description in the beacon api spec is just copy and paste from v1 with no reference to v1 https://github.com/ethereum/beacon-APIs/blob/860584e71109cc9dc34ab6b67c646ae9a818c07e/apis/beacon/pool/attestations.v2.yaml#L58